app/LoadedApp/LoadedApp.tsx

```diff
@@ -850,9 +855,18 @@ class LoadedAppClass extends Component<LoadedAppClassProps, AppStateLoaded> {
   };
 
   onClickOKServerWallet = async () => {
-    const { info } = this.state;
+    const beforeCurrencyName = this.state.info.currencyName;
 
     if (this.state.newServer) {
+      const resultStr: string = await RPCModule.execute('changeserver', this.state.newServer);
```

Comment:
`changeserver` is executed twice? 
https://github.com/zingolabs/zingo-mobile/pull/306/files#diff-bf72cf64f7dea214b172848ad9b6d12d138a4a82098cce4802c6b2ebdf900c52L856

Reply:
fixed!